Gemfile

```diff
@@ -5,8 +5,8 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in rspec_honeycomb_formatter.gemspec
 gemspec
 
+gem 'github_changelog_generator'
```

Comment:
I realise this isn't your change, but I was wondering if we should point to a specific build of this gem like we do in our modules? E.g. https://github.com/puppetlabs/puppetlabs-hocon/blob/master/Gemfile#L32

Reply:
This repo is not using module-labelling  conventions.